recipes/treelite/bld.bat

```diff
@@ -0,0 +1,12 @@
+mkdir build
+cd build
+cmake .. -G "NMake Makefiles"
+if errorlevel 1 exit 1
+
+nmake
+if errorlevel 1 exit 1
+
+cd ..
+cd python
+python setup.py install
```

Comment:
You might be able to use `pip` here (and in the `bld.bat` as in:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L35

but be sure to add `pip` to `host` if you do.